workspace/utilities/master/favicon.xsl

```diff
@@ -2,12 +2,25 @@
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
 
 <xsl:template name="favicon">
-	<link rel="shortcut icon"     href="{$root}/favicon.ico" type="image/vnd.microsoft.icon" />
-	<link rel="icon"              href="{$root}/favicon.ico" type="image/ico" />
-	<link rel="icon"              href="{$root}/favicon.png" type="image/png" />
-	<link rel="apple-touch-icon"  href="{$root}/favicon.png" type="image/png" />
-	<link rel="pavatar"           href="{$root}/favicon.png" type="image/png" />
-	<meta name="msapplication-TileImage" content="{$root}/favicon.png" />
+
+	<xsl:variable name="filename">
+		<xsl:choose>
+			<xsl:when test="$debug = true()">
+				<xsl:text>favicon-dev</xsl:text>
+			</xsl:when>
+			<xsl:otherwise>
+				<xsl:text>favicon</xsl:text>
+			</xsl:otherwise>
+		</xsl:choose>
+	</xsl:variable>
```

Comment:
1. We could simplify the logic and :up: DRY.
1. We should also check for `dev` instead of `debug`

```xslt
<xsl:variable name="filename">
	<xsl:text>favicon</xsl:text>
	<xsl:if test="$dev = true()">
		<xsl:text>-dev</xsl:text>
	</xsl:if>
</xsl:variable>
```